.travis.yml

```diff
@@ -1,14 +0,0 @@
-dist: xenial
-sudo: required
-
-language: python
-
-python:
-  - "3.7"
-
-before_script:
-  - sudo apt-get install libxml2-utils
-  - pip install --upgrade setuptools
```

Comment:
I don't see this in the new script. Is it necessary?

Reply:
Good question; I thought setuptools would be sufficiently up-to-date but perhaps not. Have added this to lint.yaml.